src/Compilers/CSharp/Portable/xlf/CSharpResources.cs.xlf

```diff
@@ -8730,6 +8725,11 @@ Pokud chcete odstranit toto varování, můžete místo toho použít /reference
         <target state="new">Invalid object creation</target>
         <note />
       </trans-unit>
+      <trans-unit id="FTL_InvalidInputFileName">
```

Comment:
Is it possible to keep the existing translations for these, since all that was changed was the id of the string? Or does it have to be "retranslated"? cc @tmeschter

Reply:
I don't think there's an _automatic_ way to keep the existing translation under a new ID, but it isn't a big deal. On their next pass, the Loc team will find the existing translation in their master database and reuse that for the "new" resource--no real work by a human will be required.